backend/Epicenter.Application/Controllers/Delegates/RecognitionDelegate.cs

```diff
@@ -31,11 +36,14 @@ public async Task<List<RecognizedObject>> GetRecognitionResultsAsync(string imag
             }
             catch (Exception ex)
             {
-                throw ex;
+                _logger.Log(LogType.ERROR, ex.Message);
+                throw;
             }
-            List<RecognizedObject> responses = plateResponse
+            RecognizedObject[] responses = plateResponse
                 .Concat(personResponse)
-                .ToList();
+                .ToArray();
+            _logger.Log(LogType.NORMAL, MessageBuilder.BuildResponseMessage(responses));
+            _logger.Log(LogType.NORMAL, "Recognition finished");
```

Comment:
change the message to "Recognition finished successfully" pls, will be easier to differentiate try-catch and proper control flow